<launch>
  <node pkg="openkite" type="ekf_node" name="ekf_node">
    <param name="kite_params" value="/home/johannes/catkin_ws/src/openkite/data/umx_radian.yaml" />
  </node>
  <node pkg="openkite" type="optitrack_client" name="optitrack" output="screen">
    <param name="server" value="192.168.1.100"/>
  </node>
  <node pkg="openkite" type="control_proxy_node" name="control_proxy" output="screen" />
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/cu.usbserial-AL022K69"/>
    <param name="baud" value="250000"/>
  </node>
</launch>
